xc/xenops_server_xen.ml

```diff
@@ -3074,6 +3074,9 @@ let init () =
 				info "Written %s to disable the hotplug/udev scripts" disable_udev_path;
 			)
 	end;
+	
+	Device.Dm.init_qmp_event ();
```

Comment:
no need (see above).

Reply:
I think we need, see my answer above, thanks.